services/UserAccountService.ts

```diff
@@ -192,4 +194,55 @@ export default class UserAccountService {
       return userProfile;
     });
   }
+
+  /**
+   * This method is used to update the access levels of multiple users at once.
+   * @param accessUpdates - an array of objects containing the user and their new access level
+   * @param emails - an array of emails to update
+   * @param currentUser - the user making the request
+   * @returns {Promise.<UserModel[]>} - an array of updated users
+   */
+  public async updateUserAccessLevels(accessUpdates, emails: string[], currentUser: UserModel): Promise<PrivateProfile[]> { //TODO: figure out return type
+    return this.transactions.readWrite(async (txn) => {
+      // Strip out the user emails & validate that users exist
+      const userRepository = Repositories.user(txn);
+      const users = await userRepository.findByEmails(emails);
+      const emailsFound = users.map((user) => user.email);
+      const emailsNotFound = emails.filter((email) => !emailsFound.includes(email));
+
+      if (emailsNotFound.length > 0) {
+        throw new BadRequestError(`Couldn't find accounts matching these emails: ${emailsNotFound}`);
+      }
+
+
+      const updatedUsers = await Promise.all(accessUpdates.map(async (accessUpdate, index) => {
+        const { user, newAccess } = accessUpdate;
+        const { accessType } = newAccess;
+        const currUser = await userRepository.findByEmail(user);
+        // Prevent anyone from promoting user to admin
```

Comment:
I believe we are allowing admins to update users to admins... Let's double check this in the next meeting

Reply:
Any updates from this?

Reply:
uhhhh no

Reply:
have u asked alex?